.gitignore

```diff
@@ -4,7 +4,6 @@
 # Cached files from the test/benchmark runner
 monument/test/.last-results.toml
 monument/test/.pinned-benches.toml
-monument/test/.last-benches.toml
```

Comment:
I think we should rename the current `.last-benches.toml` file rather than removing this from the `.gitignore`.  The `.last-benches.toml` is overwritten every time the benches get run, so tracking it with git is likely to cause some horrific merge conflicts...

We basically just want another file to enforce a rough ordering on the benchmarks, so we should probably make it completely different to the others (which have their own purposes).  I'm not sure what to name this file, though, maybe a non-hidden file with a name like `baseline-benches.toml`?